google-api-client/pom.xml

```diff
@@ -110,7 +111,6 @@
     <dependency>
       <groupId>com.google.http-client</groupId>
       <artifactId>google-http-client-gson</artifactId>
-      <scope>test</scope>
```

Comment:
JsonFactory is an interface though, so which implementing class should I use?  It seems that any class I use adds a dependency?